R/utilities.R

```diff
@@ -4517,23 +4474,78 @@ classify_dlbcl_chapuy <- function(
         as.data.table %>%
         setkey(chrom, start, end)
 
-    # If no SV data is provided, get the SVs from GAMBL
-    if(missing(sv_data)){
-        message("No SV data is provided.")
-        message("Will retreive SVs available through GAMBL.")
 
-        sv_data <- get_combined_sv(
-            sample_ids = these_samples_metadata$sample_id,
-            oncogenes = chapuy_features$sv_features,
-            projection = projection
-        )
+    # Now use this data to classify the samples according to one of the systems
+    if(method=="chapuy"){
+
+      # SVs are needed only for Chapuy predictions
+      # If no SV data is provided, get the SVs from GAMBL
+      if(missing(sv_data)){
+          message("No SV data is provided.")
+          message("Will retreive SVs available through GAMBL.")
+
+          sv_data <- get_combined_sv(
```

Comment:
This is using a function that doesn't have the complete SV data (last updated in October). We should address this

Reply:
User can provide their own SV data in bed format, this is just resorting to use what is available if nothing else is given.

Reply:
I took a look how this will compare vs when get_manta_sv is used. There is a bit of a switch. I think I will switch this to manta since this will probably be more up to date going forward and the user can supply the output of combined sv if they wish

Reply:
This is now switched to the manta svs :heavy_check_mark: